Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:8-stretch
```

Comment:
I thought this image already had stretch by default, but it was trying to fetch jessie packages and failing on my machine, so have made this explicit.